src/common/models/transaction-details.model.ts

```diff
@@ -0,0 +1,37 @@
+/* istanbul ignore file */
+
+import { ApiProperty } from '@nestjs/swagger';
+import { TransactionStatus } from '@polymeshassociation/polymesh-sdk/types';
+
+import { FeesModel } from '~/common/models/fees.model';
+import { PayingAccountModel } from '~/common/models/paying-account.model';
+
+export class TransactionDetailsModel {
+  @ApiProperty({
+    description: 'Transaction status',
+    enum: TransactionStatus,
+    example: TransactionStatus.Idle,
+  })
+  readonly status: string;
+
+  @ApiProperty({ description: 'Transaction fees', type: FeesModel })
+  readonly fees: FeesModel;
+
+  @ApiProperty({ type: 'boolean', example: true })
```

Comment:
description field?

It should mention not every transaction can be subsidized and this can be used to check